jcl/src/java.base/share/classes/java/lang/invoke/VarHandle.java

```diff
@@ -345,7 +345,7 @@ MethodType accessModeType(Class<?> receiver, Class<?> type, Class<?>... args) {
 	VarForm vform = null;
 /*[ENDIF] Java14 */
 	
-	private final MethodHandle[] handleTable;
+	MethodHandle[] handleTable;
```

Comment:
This is set from `java.lang.invoke.IndirectVarHandle`. So, it can no longer be private or final. We can possibly add a protected method to set it, which will allow us to keep it private. Also, we can tag these changes with Java 15+ to avoid perf impact on previous versions.